recipes/mne-icalabel/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mne-icalabel" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6d8b9aced23ed936791d47322b5c998a14e5340fa7f7709b050fa776b1c0673e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools >=46.4.0
+    - wheel
+  run:
+    - python
+    - importlib-resources  # [py<39]
+    - mne-base >=1.0
+    - numpy >=1.16.0
+    - scipy >=1.2.0
+    - pytorch  # [not win]  https://github.com/conda-forge/pytorch-cpu-feedstock/issues/32
```

Comment:
Does this package works without pytorch on Windows? If not we are producing a broken package on Windows here and it would be better to make it noarch and let Windows fail the installation. (When pytorch is available this package would be ready on Windows already as noarch.)

Reply:
Wait, I cannot make it `noarch` because we have:

```
importlib-resources  # [py<39]
```

Reply:
I've now made `importlib-resources` a dependency on all platforms even though we actually only need it for `py<39`, but this allows us to make the package `noarch`

[e5b8000](https://github.com/conda-forge/staged-recipes/pull/18887/commits/e5b80008fe4c8f6392fe4b9244c03301a6d9812b)

Reply:
It is fine to have `importlib-resources` in all platforms. In theory, of things work as they are supposed to, it will default to the standard lib in places were that is available.